article/sections/_4_graph_construction.tex

```diff
@@ -0,0 +1,109 @@
+\section{(Initial) Causal Graph Construction}
+\label{sec:graph-construction}
+
+Construction of an initial causal graph typically proceeds as follows.
+At a high level, we
+\begin{enumerate}
+   \item adopt a population perspective,
+   \item brainstorm all variables that we think affect the system that generates our observations,
+   \item remove any variables that could cause bias in our causal inferences,
+   \item connect all variables in our graph according to our a-priori beliefs about causal relations amongst them
+   \item consider how the graph structure may differ between individuals and subgroups within the population.
+\end{enumerate}
+The following paragraphs describe these steps in detail.
+
+To begin, we adopt the position of a researcher concerned about population level relationships.
+This means we will think through what is a likely generative model for all individuals.
+Later in this section, we will devote time to thinking about how subgroups and individual heterogeneity may affect our causal graphs.
+
+Now, we add our first variable(s) to our graph, the outcome variable(s) of interest in our problem.
```

Comment:
Add note that we should consider relations and dependencies between these outcomes (i.e. edges between them. Reference back to activity based modeling researchers. Reference medical researchers who have data on multiple health measures. Note companies with multiple metrics for the business.)

Reply:
See edits in #165.